### PR TITLE
Check if restore has cache folder (new backup/restore version)

### PR DESCRIPTION
**Pull request by fernandog**
https://github.com/SiCKRAGETV/sickrage-issues/issues/665#issuecomment-75041585

### CODE REVIEW

**Comment by fernandog** [comment]
@abeloin is this ok? the srcDir is already defined there?

File: sickbeard/__init__.py [hunk 1]
```diff
@@ -634,7 +634,7 @@ def initialize(consoleLogging=True):
 
         # Check if we need to perform a restore of the cache folder
         restoreDir = os.path.join(DATA_DIR, 'restore')
-        if os.path.exists(restoreDir):
+        if os.path.exists(restoreDir) and os.path.exists(srcDir):
```
Comment (abeloin) [inline sickbeard/__init__.py]: This wont work as you intend to as the srcDir is define in the function and not accesible from outside.

It should be like this:
```
if os.path.exists(restoreDir) and os.path.exists(os.path.join(restoreDir, 'cache')):
```

**Comment by fernandog** [comment]
is it ok now or should I do just on commit?

**Comment by abeloin** [comment]
That should be fine

**Comment by fernandog** [comment]
keep this close?

**Comment by abeloin** [comment]
Yes it's included in #1358

**Pull request closed without merging**